setuptools_grpc/build_grpc.py

```diff
@@ -0,0 +1,69 @@
+"""Plugin for setuptools providing python gRPC modules build."""
+import os
+from distutils import log
+from glob import glob
+
+import pkg_resources
+from grpc_tools.protoc import main as protoc_main
+from setuptools import Command
+
+from ._utils import grpc_py_module_name, listify_value, py_module_name
+
+__all__ = ['build_grpc']
+
+
+class build_grpc(Command):
+    """Custom command that builds protobuf and gRPC python modules."""
+
+    description = 'build grpc script'
+    user_options = [
+        ('proto-files=', None, 'newline separated list of glob patterns matching protobuf files'),
+        ('grpc-files=', None, 'newline separated list of glob patterns matching grpc files'),
+        ('proto-path=', None, 'path to directory with protobuf files'),
+        ('output-path=', None, 'output path'),
+    ]
+
+    def initialize_options(self):
+        self.proto_files = []
+        self.grpc_files = []
+        self.proto_path = '.'
+        self.output_path = '.'
+
+    def finalize_options(self):
+        self.ensure_dirname('proto_path')
+        self.ensure_dirname('output_path')
+
+        proto_files, grpc_files = [], []
+
+        for fileglob in filter(None, listify_value(self.proto_files, '\n')):
```

Comment:
I think list comprehensions are preferred over `filter` function.

Reply:
I find `filter` easier to understand here. Comparable list comprehension would look like this:

```python
for fileglob in [value for value in listify_value(self.proto_files, '\n') if bool(value)]:
```